tests/test_llm_simple_service_streaming.py

```
@@ -128,6 +128,62 @@ def test_build_payload_includes_top_p_when_provided():
+                            ],
+                        },
+                        {"token": "skip", "logprob": "bad"},
+                    ]
+                }
+            }
+        ]
+    }
+    telemetry = llm_simple_service._extract_logprobs_telemetry(payload)
+    assert telemetry is not None
+    assert telemetry["kind"] == "logprobs"
+    assert telemetry["content"][0]["token"] == "A"
```

Ruff (0.15.13)

[error] Possible hardcoded password assigned to: "token"

(S105)